### PR TITLE
smart contracts > maintain > control

### PR DESCRIPTION
**Pull request by domwoe**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/developer-docs/smart-contracts/maintain/canister-control.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Control
+
+## Overview
+
+A canister has a basic permission model of a list of controllers which have administrative rights over the canister. A controller is specified by a principal, which can be self-authenticating, e.g. a `dfx` developer identity, or another canister. A canister can have multiple controllers, and each controller has the same administrative rights over the canister. A canister can also have no controller, in which case the canister becomes immutable and cannot be upgraded or deleted. 
+
+## Setting the controllers of a canister
+
+The controllers of a canister are set when the canister is created. If no explicit controller settings are specified, the default controller is the principal that created the canister. If the canister is created with `dfx` then the default controller list contains the developer identity and the principial of the associated cycles wallet canister.
+
```
Comment (jennifertrin) [inline docs/developer-docs/smart-contracts/maintain/canister-control.md:10]: How would you define non-default controllers upon deployment?

File: docs/developer-docs/smart-contracts/maintain/canister-control.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Control
+
+## Overview
+
+A canister has a basic permission model of a list of controllers which have administrative rights over the canister. A controller is specified by a principal, which can be self-authenticating, e.g. a `dfx` developer identity, or another canister. A canister can have multiple controllers, and each controller has the same administrative rights over the canister. A canister can also have no controller, in which case the canister becomes immutable and cannot be upgraded or deleted.
```
[the remark this answers is not in the document]
Comment (domwoe) [reply docs/developer-docs/smart-contracts/maintain/canister-control.md:5]: Yep, I can do that.

File: docs/developer-docs/smart-contracts/maintain/canister-control.md (hunk 1 above)
Comment (domwoe) [reply docs/developer-docs/smart-contracts/maintain/canister-control.md:10]: I don't t think this is used in practice a lot (except you do it programmatically from a canister) which I didn't want to discuss at this point.

Comment (jennifertrin) [reply docs/developer-docs/smart-contracts/maintain/canister-control.md:10]: Ah, so you are saying when a canister creates another canister?

Comment (domwoe) [reply docs/developer-docs/smart-contracts/maintain/canister-control.md:10]: Yes, exaclty.

**Pull request merged**